README.md

```diff
@@ -1,2 +1,25 @@
 # line-bot
 Line bot for our random features
+
+## Instructions to get started:
+
+Requirements: 
+* Python 3.6+ 
+* pipenv
```

Comment:
you should put python package requirements and node requirements in a separate section.